src/easydmp/dmpt/models.py

```diff
@@ -1510,6 +1515,12 @@ def frame_canned_answer(self, answer, frame=True):
             result = self.framing_text.format(answer)
         return mark_safe(result)
 
+    def is_optional_section_question(self):
+        """
+        True if this question is the auto-generated question about answering a section or not
+        """
+        return self.position == 0
+
```

Comment:
Good thing to have, but not in the right location. I'll reorganize this myself later, put all the "is_"-methods together.

Reply:
Yes, to not spread the detail of the index being 0 around everywhere.

Reply:
Exactly!